### DISCUSSION TITLE
Simply the best

### DISCUSSION THREAD

**Discussion by ayoung19** [mantinedev/mantine#7315] [Kind words]
A Chrome extension I recently built (https://github.com/ayoung19/clipboard-history) is made up of _only_ Mantine components. Not only has it been a breeze to develop (I don't remember ever having to fight Mantine), but a common piece of positive feedback I receive is how much better the UI is compared to alternative extensions! Mantine has truly been a game-changer and I appreciate all the work that goes into it. If my extension ever generates any revenue, a @mantinedev sponsorship is the first thing it'll be spent on!

**Comment by rtivital** [comment]
Thanks for kind words!
